README.md

```
@@ -1,194 +1,379 @@
 <p align="center">
-  <img src="https://raw.githubusercontent.com/SirLordPouya/AndroidAppUpdater/master/pics/icon.png" width="250">
+  <img src="https://raw.githubusercontent.com/HeyPouya/AndroidAppUpdater/master/pics/icon.png" width="250">
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)